bin/live/pycbc_live_supervise_single_significance_fits

```diff
@@ -270,9 +188,9 @@ def do_fitting(args, day_dt, day_str):
 
         daily_options['gps-start-time'] = f'{gpstime.utc_to_gps(day_dt).gpsSeconds:d}'
         daily_options['gps-end-time'] = f'{gpstime.utc_to_gps(day_dt + timedelta(days=1)).gpsSeconds:d}'
-        daily_args += dict_to_args(daily_options)
+        daily_args += sv.dict_to_args(daily_options)
 
-        run_and_error(daily_args, controls)
+        sv.run_and_error(daily_args)
```

Comment:
Doesn't this line still need to pass `controls` ? 
`sv.run_and_error` takes two arguments

Reply:
I've realised that the supervision scripts need combining into one anyway, so this will be fixed when that is done